dynamic-course-demo/manifests/02-sonataflow_dynamic-course-select.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlow
+metadata:
+  annotations:
+    sonataflow.org/description: Simple workflow demonstrating basics of using the
+      orchestrator-form-widgets library. Requires corresponding HTTP endpoints to
+      be up and proxy to be configured.
+    sonataflow.org/expressionLang: jq
+    sonataflow.org/profile: dev
```

Comment:
Are you intentionally using the `dev` profile?
We should be better with gitops provide for our needs.
Which means we should build the image.

Reply:
perhaps for now, preview will be better since it will use the DI instead, while in dev mode it uses the embedded one.